packages/docs-reanimated/docs/threading/runOnUI.mdx

```diff
@@ -71,6 +71,8 @@ import RunOnUISrc from '!!raw-loader!@site/src/examples/RunOnUI';
 
 - Make sure not to execute `runOnUI` on the UI thread as this will result in an error.
 
+- On the Web, `runOnUI` immediately calls the function given as the argument. In browsers there's no UI thread available.
```

Comment:
Actually, `runOnUI` on web behaves just like `requestAnimationFrame` – it creates a version of the function that, when called, will be scheduled to be run with given arguments on next animation frame.

Reply:
thanks! fixed in [3e8f977](https://github.com/software-mansion/react-native-reanimated/pull/6737/commits/3e8f97740c25d66afc1e615af832f1ddab59df2d)